docs/ref/forms/api.txt

```diff
@@ -934,6 +934,29 @@ When set to ``True`` (the default), required form fields will have the
 ``use_required_attribute=False`` to avoid incorrect browser validation when
 adding and deleting forms from a formset.
 
+.. attribute:: Form.bound_field_class
+
+Allows you set a different :class:`~django.forms.BoundField` class to use for the form. This
+allows to override, for instance, :meth:`~django.forms.BoundField.label_tag`/
+:meth:`~django.forms.BoundField.legend_tag` to use in templates.
```

Comment:
Maybe:

"As an example, this allows overriding `label_tag` or `legend_tag` which are used by the default form rendering in templates:"

Also, further down the form field API docs mention customizing bound fields; the introduction of the "Customizing BoundField" paragraph should possibly be reworded to explicitly mention that this is a way to customize the BoundField class per-field? It's probably obvious from the context but I wanted to point it out.

Reply:
> Also, further down the form field API docs mention customizing bound fields

You mean [this section](https://github.com/django/django/blob/main/docs/ref/forms/api.txt#L1147)?

Reply:
No, further below: 

https://github.com/django/django/blob/c2c7dbb2f88ce8f0ef6d48a61b93866c9926349a/docs/ref/forms/api.txt#L1444-L1483

Reply:
Done.

Reply:
Ok done. Is that ok?

Reply:
I think it's great, thank you.